tag.go

```diff
@@ -43,8 +51,34 @@ type htmler interface {
 	HTML() template.HTML
 }
 
+func parseTagEmbed(b interface{}, bb *bytes.Buffer) {
+	switch tb := b.(type) {
+	case htmler:
+		bb.Write([]byte(tb.HTML()))
+	case fmt.Stringer:
+		bb.WriteString(tb.String())
+	case interfacer:
+		val := tb.Interface()
+		if tb.Interface() == nil {
+			val = ""
+		}
+
+		bb.WriteString(fmt.Sprint(val))
+	default:
+		bb.WriteString(fmt.Sprint(tb))
+	}
+}
+
 func (t Tag) String() string {
 	bb := &bytes.Buffer{}
+
+	if len(t.BeforeTag) > 0 {
```

Comment:
You don't need this `if` statement. The `range` will only run if there anything in the slice.